README.md

```diff
@@ -1,97 +1,58 @@
 
-This [ESPHome](https://esphome.io) package creates a local network connection to econet-based device, like a Rheem Heat Pump Water Heater (HPWH), and creates entities in Home Assistant to control and monitor these devices.  This package provides more detailed and reliable sensors than Rheem's cloud-based [econet integration](https://www.home-assistant.io/integrations/econet/) available in Home Assistant.  This package and the Rheem econet integration can, however, coexist if desired.
+This [ESPHome](https://esphome.io) package enables local control of a Rheem appliance, like a Rheem Heat Pump Water Heater (HPWH), and creates entities in Home Assistant to control and monitor these devices.  This package provides more detailed and reliable sensors than Rheem's cloud-based [Econet integration](https://www.home-assistant.io/integrations/econet/) available in Home Assistant, and does so without any requirement for internet access by accessing the device via it's RJ11 diagnostics port.
 
-## Supported Rheem Hardware
-
-Rheem Heat Pump Water Heaters, Tankless Water Heaters, Electric Tank Water Heaters, and HVAC Systems are supported.
-
-## Suggested Hardware Setup
-
-For a simple and proven platform on which to run this package you will need an RS485 interface and an ESP32 board. These are available as follows:
+This package and the Rheem econet integration can coexist if desired.
 
-- One M5Stack ATOM RS485 K045 Kit: [mouser](https://www.mouser.com/ProductDetail/M5Stack/K045?qs=81r%252BiQLm7BQ2ho0A5VkoNw%3D%3D) has them while stock is available. M5Stack has discontinued the combined K045 Package.
-- Or, the same thing, but ordered as two separate parts: The M5Stack *A131 RS485 Base* [M5Stack A131](https://shop.m5stack.com/products/atomic-rs485-base) and the M5Stack *C008 Atom Lite* [M5Stack C008](https://shop.m5stack.com/products/atom-lite-esp32-development-kit?variant=32259605200986).
-  - Both Digikey and Mouser have the Atom. The RS485 Base is not available from either vendor as of this update.
-    - [Digikey C008](https://www.digikey.com/en/products/detail/m5stack-technology-co-ltd/C008/12088545) (Never mind the image Digikey displays, see the datasheet)
-    - [Mouser C008](https://www.mouser.com/ProductDetail/M5Stack/C008?qs=sGAEpiMZZMuqBwn8WqcFUj1SFkunHY10TJ3jnDGC7E4NSjPubczP2Q%3D%3D)
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+<p float="left">
+    <img src="https://github.com/esphome-econet/econet-docs/blob/main/photos-and-screenshots/Device_Page.png?raw=true" alt="Example Home Assistant device page for an ESPHome-econet device" width=40%>
```

Comment:
Do you want to blur your mac and serial number?
Can you get new screenshots without all the additional sensors you have (such as the ATOM ones) that aren't included with the default installation. This is to avoid confusion with users wondering how come this screenshot has more sensors than they ones they will get.
It would be good to take new screenshots to include recent changes, mostly around vacation mode.